jerry-libm/CMakeLists.txt

```diff
@@ -18,11 +18,17 @@ project (${JERRY_LIBM_NAME} C)
 
 # Compiler / linker flags
 # TODO: Reduce the below list of warning/error disablings as much as possible
-set(COMPILE_FLAGS_LIBM "${COMPILE_FLAGS_LIBM} -Wno-error=sign-compare")
-set(COMPILE_FLAGS_LIBM "${COMPILE_FLAGS_LIBM} -Wno-error=sign-conversion")
-set(COMPILE_FLAGS_LIBM "${COMPILE_FLAGS_LIBM} -Wno-sign-conversion")
-set(COMPILE_FLAGS_LIBM "${COMPILE_FLAGS_LIBM} -Wno-sign-compare")
-set(COMPILE_FLAGS_LIBM "${COMPILE_FLAGS_LIBM} -Wno-strict-aliasing")
+if(USING_MSVC)
+  set(COMPILE_FLAGS_LIBM "${COMPILE_FLAGS_LIBM} /wd4146")
+  set(COMPILE_FLAGS_LIBM "${COMPILE_FLAGS_LIBM} /wd4018")
+  set(COMPILE_FLAGS_LIBM "${COMPILE_FLAGS_LIBM} /wd4293")
+else()
+  set(COMPILE_FLAGS_LIBM "${COMPILE_FLAGS_LIBM} -Wno-error=sign-compare")
+  set(COMPILE_FLAGS_LIBM "${COMPILE_FLAGS_LIBM} -Wno-error=sign-conversion")
+  set(COMPILE_FLAGS_LIBM "${COMPILE_FLAGS_LIBM} -Wno-sign-conversion")
+  set(COMPILE_FLAGS_LIBM "${COMPILE_FLAGS_LIBM} -Wno-sign-compare")
+  set(COMPILE_FLAGS_LIBM "${COMPILE_FLAGS_LIBM} -Wno-strict-aliasing")
+endif()
```

Comment:
If jerry-libm gets disabled for MSVC by default, this change will become unnecessary.